app/views/teams/_form.html.erb

```diff
@@ -27,6 +27,17 @@
     </div>
   </div>
 
+  <div class="form-group row">
+          <%= f.label :avatar, t('activerecord.models.image.one'), class: 'col-form-label col-lg-2'%>
+          <div class="col-lg-10">
+            <%= image_tag @team.avatar_url || image_path('missing_avatar.png'), id: "preview", style: "max-width:100px; max-height:100px"%>
+            <%= f.file_field :avatar, class: "form-control-file" %>
+            <% if @team.avatar %>
+                  <%= t('helpers.submit.delete', model: t('activerecord.models.image.one'))+ "  " %><%= f.check_box :remove_avatar %>
+              <% end %>
+    </div>
+    </div>
```

Comment:
Please double-check html structure.

- Label "Bild" is not formatted correctly
- Checkbox is not visible

<img width="1440" alt="screen shot 2018-01-15 at 14 49 53" src="https://user-images.githubusercontent.com/18228544/34945393-7e180726-fa03-11e7-96e8-bf427495ff97.png">